modules/rhel-compute-overview.adoc

```diff
@@ -6,30 +6,20 @@
 [id="rhel-compute-overview_{context}"]
 = About adding RHEL compute nodes to a cluster
 
-In {product-title} {product-version}, you have the option of using Red Hat
-Enterprise Linux (RHEL) machines as compute, or worker, machines in your
-cluster if you use a user-provisioned infrastructure installation. You
-must use {op-system-first} machines for the control plane, or
-master, machines in your cluster.
+In {product-title} {product-version}, you have the option of using Red Hat Enterprise Linux (RHEL) machines as compute, or worker, machines in your cluster if you use a user-provisioned infrastructure installation. You must use {op-system-first} machines for the control plane, or master, machines in your cluster.
 
-As with all installations that use user-provisioned infrastructure, if you
-choose to use RHEL compute machines in
-your cluster, you take responsibility for all operating system life cycle
-management and maintenance, including performing system updates, applying
-patches, and completing all other required tasks.
+As with all installations that use user-provisioned infrastructure, if you choose to use RHEL compute machines in your cluster, you take responsibility for all operating system life cycle management and maintenance, including performing system updates, applying patches, and completing all other required tasks.
 
 [IMPORTANT]
 ====
-Because removing {product-title} from a machine in the cluster requires
-destroying the operating system, you must use dedicated hardware for any
-RHEL machines that you add to the cluster.
+Because removing {product-title} from a machine in the cluster requires destroying the operating system, you must use dedicated hardware for any RHEL machines that you add to the cluster.
 ====
 
 [IMPORTANT]
 ====
-Swap memory is disabled on all RHEL machines that you add to your {product-title}
-cluster. You cannot enable swap memory on these machines.
+Swap memory is disabled on all RHEL machines that you add to your {product-title} cluster. You cannot enable swap memory on these machines.
 ====
 
-You must add RHEL compute machines to the cluster after you initialize the
-control plane.
\ No newline at end of file
+You must add RHEL compute machines to the cluster after you initialize the control plane.
+
+RHEL compute machines are not included when your cluster entitlement is calculated.
```

Comment:
Really?  How is support for these nodes funded?

Reply:
That's what they said in the what's new presentation.

Reply:
I double-checked with Tim. You have to entitle the RHEL nodes for RHEL, but they really don't count against your OCP entitlements. I've updated this statement per our discussion on Slack.